kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/InCommitTimestampSuite.scala

```diff
@@ -403,12 +403,15 @@ class InCommitTimestampSuite extends DeltaTableWriteSuiteBase {
       verifyTableProperties(
         tablePath,
         ListMap(
+          // appendOnly, invariants implicitly supported as the protocol is upgraded from 2 to 7
           IN_COMMIT_TIMESTAMPS_ENABLED.getKey -> true,
+          "delta.feature.appendOnly" -> "supported",
           "delta.feature.inCommitTimestamp" -> "supported",
+          "delta.feature.invariants" -> "supported",
```

Comment:
these are fake properties populated by delta-spark right? can you add a comment this was not clear w/o looking at the implementation for `verifyTableProperties`

Reply:
or maybe `verifyTableProperties` is just poorly named..

Reply:
added comment. Yeah, this utility needs cleanup.